manifests/s/SWI-Prolog/SWI-Prolog/9.0.0/SWI-Prolog.SWI-Prolog.locale.en-US.yaml

```diff
@@ -0,0 +1,238 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SWI-Prolog.SWI-Prolog
+PackageVersion: 9.0.0
+PackageLocale: en-US
+Publisher: SWI-Prolog
+PublisherUrl: https://www.swi-prolog.org/
+PublisherSupportUrl: https://www.swi-prolog.org/community.html
+# PrivacyUrl:
+Author: Jan Wielemaker
+PackageName: SWI-Prolog
+PackageUrl: https://www.swi-prolog.org/
+License: BSD-2-Clause
+LicenseUrl: https://www.swi-prolog.org/license.html
+Copyright: Copyright (c) 2022, SWI-Prolog Solutions b.v. All rights reserved.
+# CopyrightUrl:
+ShortDescription: A versatile implementation of the Prolog language
+Description: SWI-Prolog offers a comprehensive free Prolog environment. Since its start in 1987, SWI-Prolog development has been driven by the needs of real world applications. SWI-Prolog is widely used in research and education as well as commercial applications.
+Moniker: swipl
+Tags:
+- language
+- logic
+- programming
+- prolog
+# Agreements:
+ReleaseNotes: |-
```

Comment:
These release notes need updating

Reply:
I guess they're too lengthy to be actually useful and we should probably get rid of them? To anyone who wants to read the whole changelog, the ReleaseNotesUrl may suffice.